src/olympia/conf/dev/settings.py

```diff
@@ -53,7 +53,7 @@
 
 SERVICES_DATABASE = get_db_config('SERVICES_DATABASE_URL')
 
-SLAVE_DATABASES = ['slave']
+REPLICA_DATABASES = ['slave']
```

Comment:
@eviljeff do we also need to change `slave` to replica?

Reply:
it's not needed to support newer versions of django-multi-db, (`slave` refers to the key in a setting we define) but it would be more consistent.  If you do you need to change the dict key a few lines above (line 50 in this file) also to `'replica'`